learnbase/src/main/webapp/css/style.css

```diff
@@ -860,3 +860,4 @@ iframe {
   0% { transform: rotate(0deg); }
   100% { transform: rotate(360deg); }
 }
+
```

Comment:
Change entire style to use this format:

```
xxx xxx xxx {
  xxx: xxx,
  ...
}
```

Instead of putting selectors/classes on a separate line